backend/blueprint/api/controllers/loginUser.py

```diff
@@ -45,3 +47,36 @@ def login():
     return jsonify(
         Response(403).generateMessage(
             'No name key received'))
+
+
+@router.route('/get_user', methods=['GET'])
+def index():
+    data = request.args
+    user = User.getUser(username=data.get('username'))
+    schema = UserSchema()
+    return jsonify(schema.dump(user).data)
+
+
+@router.route('/register', methods=['POST'])
```

Comment:
@gabru-md why are maintaining two versions of register, why are we not using UserSchema in registerUser.py?

Reply:
I will delete that asap

Reply:
we will be using the one in loginUser.py or maybe I can change the code from loginUser to RegisterUSer